readme.md

```diff
@@ -60,6 +60,13 @@ You can configure `chimi` using a configuration file, it might be a JSON or Java
     "./config": "config",
     "es6-promise: ""
   },
+  "globals": {
+    "answer": 42,
+    "add": "(a, b) => a + b",
+    "message": "'hello world'",
+    "fruits": "['orange', 'apple']",
+    "config": "{ port: 8080 }",
```

Comment:
This might be misleading, if you use a JSON as a config file then `add`, `fruits` and `config` will be strings (and `message` will be a string with single quotes inside). The lambda can't be used in a JSON.

I think this should be:

```
"globals: {
  "answer": 42,
  "message": "hello world",
  "fruits": ["orange", "apple"],
  "config": { port: 8080 }
}
```

And maybe some clarification that if you want to add a function as a global, you need to use `chimi.config.js`.

Reply:
Oh, wait, I'm completely wrong, my bad.